.ci/release/create_bundle_manifest.yml

```diff
@@ -0,0 +1,28 @@
+---
+platform: linux
+
+image_resource:
+    type: docker-image
+    source:
+        # This task requires a container with 'jq'.
+        repository: nmckinley/jq
+        tag: 'v0.0.1'
+
+inputs:
+    - name: puppet-resourcemanager-forge
```

Comment:
I just had an idea: could we use [config](https://concourse-ci.org/task-step.html#config) instead of file for our concourse tasks, and then have these files be .yml.tmpl and get {% include %}-ed? If that worked, then we could have less hardcoding in these.